examples/templates/gcp-devcontainer/README.md

```diff
@@ -0,0 +1,62 @@
+---
+display_name: Google Compute Engine (Devcontainer)
+description: Provision a Devcontainer on Google Compute Engine instances as Coder workspaces
+icon: ../../../site/static/icon/gcp.png
+maintainer_github: coder
+verified: true
+tags: [vm, linux, gcp, devcontainer]
+---
+
+# Remote Development in a Devcontainer on Google Compute Engine
```

Comment:
Thoughts on making a basic architecture diagram that shows that there is a VM that runs the docker container? I know that is a bit of extra work but could help make it clear.

Just looked up the [Codespaces architecture](https://docs.github.com/en/codespaces/overview) and they do that.